src/shapes/Rect.ts

```diff
@@ -186,15 +200,7 @@ export class Rect extends FabricObject {
   /* _FROM_SVG_END_ */
 }
 
-export const rectDefaultValues: Partial<TClassProperties<Rect>> = {
-  type: 'rect',
-  rx: 0,
-  ry: 0,
-};
-
-Object.assign(Rect.prototype, {
-  ...rectDefaultValues,
-});
+Rect.prototype.type = 'rect';
```

Comment:
those may also end up staying here for 6.0 since those aren't either fundamental or architecturally important.
Is just a string we need for legacy reason and that may be made obsolete by  Class.name at some point.

Reply:
This is why I made #8714, I saw this coming and we can already move to Class.name in that PR.